### PR TITLE
update to the update

### PR DESCRIPTION
**Pull request by sallyom**
/cc @whitel  
/cc @umohnani8 

- added link to FAQ from index
- day of live workshops -> day of live virtual workshops
- email with any other questions -> email with any other questions and especially any ideas you have!

### CODE REVIEW

**Comment by whitel** [comment]
LGTM

**Comment by umohnani8** [comment]
LGTM

File: source/us/index.blade.php [hunk 1]
```diff
@@ -257,10 +257,13 @@
...
 <section id="participation" class="mx-auto py-5 col-lg-11">
 	<div class="row">
-		<div class="mont-700 h1 underline position-relative mx-5 pb-4">How to participate?</div>
+		<div class="mont-700 h1 position-relative mx-5 pb-4">How to participate?</div>
 	</div>
 	<div class="row py-4">
-		<h4 class="mont-700 mb-0 position-relative mx-5 pb-4">Watch this space! We'll be adding information to help you produce your videos.</h4>
+		<h4 class="mont-700 mb-0 position-relative mx-5 pb-4">Watch this space. We'll be adding information to help you produce your videos.</h4>
+    </div>
+	<div class="row py-6">
```
Comment (jridky) [inline source/us/index.blade.php:265]: py-6 class doesn't exist in bootstrap css. the class should be set to py-4

File: source/us/index.blade.php [hunk 2]
```diff
@@ -257,10 +257,13 @@
 -->
 <section id="participation" class="mx-auto py-5 col-lg-11">
 	<div class="row">
-		<div class="mont-700 h1 underline position-relative mx-5 pb-4">How to participate?</div>
+		<div class="mont-700 h1 position-relative mx-5 pb-4">How to participate?</div>
 	</div>
 	<div class="row py-4">
```
Comment (jridky) [inline source/us/index.blade.php:262]: this py-4 should be removed (only last div with class row should contain this class)

File: source/us/index.blade.php [hunk 3]
```diff
@@ -257,10 +257,13 @@
 -->
 <section id="participation" class="mx-auto py-5 col-lg-11">
 	<div class="row">
-		<div class="mont-700 h1 underline position-relative mx-5 pb-4">How to participate?</div>
```
Comment (jridky) [inline source/us/index.blade.php:260]: keep section title underlined to remain consistent with visual standards.

**Pull request merged**